app/src/shared/components/Meatball/meatball.pcss

```diff
@@ -31,8 +31,7 @@ a:--enter .root {
   fill: var(--white);
 }
 
-.white:--enter,
-a:--enter .white {
```

Comment:
@mondoreale: Do you remember why the a:--enter was included here? This breaks the hover effect when used inside Tile component because then the hover will trigger already when mouse is over the Tile and not when mouse is over the Meatball element.

Reply:
IIRC it's for situations where `<a>` comes with padding and wraps a `Meatball` element. In this case, w/o `a:--enter .white` we'll get a pointer cursor but the meatball won't change its color (when padded area is hovered, not the meatball itself).

@tumppi you can go ahead and remove it. It shouldn't block you and I feel like we can live w/o it for the time being, and make it better in the future (if it's an issue).